src/main/java/net/sf/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -947,7 +947,11 @@ public void updateAllContentSelectors() {
     @Subscribe
     public void listen(FieldChangedEvent fieldChangedEvent) {
         String newValue = fieldChangedEvent.getNewValue() == null ? "" : fieldChangedEvent.getNewValue();
-        setField(fieldChangedEvent.getFieldName(), newValue);
+        if (SwingUtilities.isEventDispatchThread()) {
```

Comment:
I assume that this `if` statement is necessary and one cannot just call `invokeLater()`?